src/cliAction/startStopDeleteProject/handler_delete.go

```diff
@@ -0,0 +1,64 @@
+package startStopDeleteProject
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/zerops-io/zcli/src/i18n"
+	"github.com/zerops-io/zcli/src/utils"
+	"github.com/zerops-io/zcli/src/zeropsApiProtocol"
+)
+
+func (h *Handler) RunDelete(ctx context.Context, config RunConfig, projectId string, actionType string) error {
+
+	isConfirmed := string(actionType[0]) == "y"
+	if !isConfirmed {
```

Comment:
This is maybe a little bit nit picky but if you name the variable just `confirmed` then it's more grammatically correct. :smile:

`if !confirmed`

Reply:
you're absolutely right, dunno why I did this  :D